### PR TITLE
Add support for channel events 

### PR DESCRIPTION
**Pull request by xdumaine**
by proxying add channel and channel message to the session. @legastero @fippo?

I'm guessing you do something in talky with `session.pc._remoteDataChannels`? That's how I was picking up the channel and events for last-n, and figured this made more sense.

### CODE REVIEW

**Comment by fippo** [comment]
we do have the addChannel -- but imo beyond that this is too app-specific to implement here (and I don't like the simplewebrtc names :-))

**Comment by xdumaine** [comment]
@fippo What if I left `addChannel` and removed `sendDirectly` and `onmessage` - that way channels are emitted the same as streams, but nothing is done `with` the channel?

**Comment by xdumaine** [comment]
Updated for simpler, less app specificity.

File: index.js [hunk 1]
```diff
@@ -88,6 +89,11 @@ Object.defineProperties(MediaSession.prototype, {
             }
             return [];
         }
+    },
+    channels: {
+        get: function () {
+            return this.pc._remoteDataChannels.concat(this.pc._localDataChannels);
```
Comment (fippo) [inline index.js]: don't use those please... they're just a workaround for both browsers garbage collecting data channels (unfixed for 1+ year now)

Comment (xdumaine) [inline index.js]: I didn't realize that. What's the proper way to access them?

Comment (fippo) [inline index.js]: oddly, there is no equivalent to getLocalStreams/getRemoteStreams. Managing the datachannel lifecycle is app reponsibility. I don't think the media session is the right place even though I agree we should be emitting channelAdded to enable the application (or any modules on top of this) to handle this.

tl;dr: can we just do onAddChannel? ;-)

Comment (xdumaine) [inline index.js]: Sounds good to me. I'll update.

**Comment by xdumaine** [comment]
@fippo Updated for just addChannel

**Comment by legastero** [comment]
+1

**Comment by fippo** [comment]
published as 1.4.0, thanks. @legastero will take care of talky :-)

**Comment by fippo** [comment]
eeek. why did we name it addChannel in talky and channelAdded here?

**Comment by xdumaine** [comment]
Oh wow that was silly.  `addChannel` is better.

**Comment by fippo** [comment]
well... neither lance now or objected. Will do minor bump. Thanks!

**Comment by fippo** [comment]
1.4.1 published
